src/plone/app/mosaic/configure.zcml

```diff
@@ -14,12 +14,16 @@
   <genericsetup:registerProfile
       name="default"
       title="Mosaic"
-      description="Installs Mosaic layout support for Plone"
+      description="Installs Mosaic layout support."
       directory="profiles/default"
       provides="Products.GenericSetup.interfaces.EXTENSION"
       post_handler="plone.app.mosaic.setuphandlers.post_handler"
       />
 
+  <utility
+      factory=".setuphandlers.HiddenProfiles"
+      name="plone.app.mosaic-hiddenprofiles" />
```

Comment:
as per ZCML style guide, the ending `/>` must be on a new line.

Reply:
we may want to change it in https://github.com/plone/bobtemplates.plone/blob/master/bobtemplates/plone/addon/src/%2Bpackage.namespace%2B/%2Bpackage.name%2B/configure.zcml.bob#L42 where I copied it from...